esi_leap/tests/objects/test_lease.py

```diff
@@ -209,37 +209,37 @@ def test_create_invalid_time(self):
             'updated_at': None
         }
 
-        c = contract.Contract(self.context, **bad_contract)
+        l = lease.Lease(self.context, **bad_lease)
 
-        self.assertRaises(exception.InvalidTimeRange, c.create)
+        self.assertRaises(exception.InvalidTimeRange, l.create)
 
     def test_create_concurrent(self):
-        c = contract.Contract(
-            self.context, **self.fake_contract)
+        l = lease.Lease(
+            self.context, **self.fake_lease)
 
-        c2 = contract.Contract(
-            self.context, **self.fake_contract)
+        c2 = lease.Lease(
```

Comment:
Should it be "l2"?

Reply:
Nice catch! This (and the others) should be fixed.